crates/common/tedge_config/src/tedge_config_location.rs

```diff
@@ -1,6 +1,6 @@
 use std::path::{Path, PathBuf};
 
-const DEFAULT_ETC_PATH: &str = "/etc";
+pub const DEFAULT_ETC_PATH: &str = "/etc/tedge";
```

Comment:
I agree with @PradeepKiruvale that this should be named with a more explanatory name. Their suggestion (`DEFAULT_TEDGE_CONFIG_PATH`) seems right!